vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -67,98 +67,79 @@
       </Annotation>
 
 
-      <!-- Metadata annotations -->
-
-      <Term Name="TemporalSupport" Type="Temporal.TemporalSupportType" AppliesTo="EntitySet NavigationProperty">
+      <Term Name="ApplicationTimeSupport" Type="Temporal.ApplicationTimeSupportType" Nullable="false" AppliesTo="Collection">
+        <Annotation Term="Core.AppliesViaContainer" />
         <Annotation Term="Core.Description">
-          <String>This entity set or navigation property supports  temporal requests</String>
+          <String>This entity set or navigation property supports temporal requests</String>
         </Annotation>
       </Term>
 
-      <ComplexType Name="TemporalSupportType">
-        <Property Name="ApplicationTime" Type="Temporal.TimeDimensionType">
-          <Annotation Term="Core.Description" String="Application time granularity" />
+      <ComplexType Name="ApplicationTimeSupportType">
+        <Property Name="CollectionKind" Type="Temporal.CollectionKindType" Nullable="false">
+          <Annotation Term="Core.Description" String="Kind of temporal collection" />

```

Comment:
Collection kind = Kind of collection. Redundant?

Reply:
This is the description of the abstract base type. Should this description list/mention all sub-types?

Reply:
Let's consider more expressive names:
* CollectionKind -> DataHistory
* PeriodKind -> UnitOfTime

Reply:
Let's try that:
- DataHistory
  - subtype DataHistoryHidden (for snapshot entity sets)
    - no properties, just a "marker type" 
  - subtype DataHistoryVisible (for time-slice entity sets)
    - properties:
      - TimeSliceStart (rename to PeriodStart?)
      - TimeSliceEnd (rename to PeriodEnd?)
      - ObjectKey
  
- UnitOfTime
  - subtype UnitOfTimeDate
    - properties:
      - ClosedClosedPeriods
  - subtype UnitOfTimeDateTimeOffset
    - properties:
      - Precision

Reply:
DataHistoryHidden -> DataHistorySnapshot because it is not really hidden, you can retrieve it with $at.
DataHistoryVisible -> DataHistoryFull